src/layer.rs

```diff
@@ -100,13 +100,20 @@ impl LayerSet {
     ///
     /// # Panics
     ///
-    /// panics if `name` is empty or if it contains any [control characters].
+    /// panics if
+    /// 1. `name` is empty or if it contains any [control characters].
+    /// 2. `name` is the default layer name but the default layer has a
+    ///    different name. Use [`Self::default_layer`] or
+    ///    [`Self::default_layer_mut`] instead.
```

Comment:
with this getting so complicated I wonder if we should just remove `get_or_create`?

Reply:
Alternatively we could just return the default layer if the user passes the default layer name, even if the default layer has another name? idk

Reply:
Hm. I made the method for two particular calls in my custom to-UFO converter thing. I suppose I could make it work without the method. Shall I remove it in this PR?

Reply:
I'll do it in a separate PR.